stories/organisms/SelectedItemBadge.stories.tsx

```diff
@@ -177,46 +178,63 @@ export const _SelectedItemBadgeMultiSelect: Story = {
         };
 
         return (
-            <div style={{ width: '200px' }}>
-                <SelectedItemBadge
-                    {...args}
-                    priority={
-                        args.priority && {
-                            ...args.priority,
-                            selectedItem: selectedPriorityItem,
-                            onChange: handlePriorityChange,
+            <>
+                <Text>
+                    Note: group selection and Select all is not correctly implemented in this
+                    example. We only showcasing the layout here.
```

Comment:
Do we still plan it to use it with current `Dropdown` as well? 🤔
If yes, I think it makes sense to try to use it there and correctly since dropdown for the badge will be changed anyway. What do you think?
If we are not planning to use it just skip the comment